scripts/tasks/build-codepen-examples.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(options) {
+  const path = require('path');
+  const transformer = require('./codepen-examples-transform');
+  const glob = require('glob');
+  const files = glob.sync(path.resolve(process.cwd(), 'src/components/**/examples/*Example*.tsx'));
+  const jscodeshift = require('jscodeshift');
+  const fs = require('fs');
+
+  // Return a promise.
+  return processFiles();
+
+  function processFiles() {
+    const promises = [];
+    if (files.length) {
+      files.forEach(file => {
+        const filePath = path.resolve(file);
+        const fileSource = fs.readFileSync(filePath).toString();
+
+        promises.push(
```

Comment:
You should utilize "async" library to do some rate limit on this instead of doing all files at once.
http://caolan.github.io/async/

Reply:
http://caolan.github.io/async/docs.html#eachLimit

Reply:
Sigh, this is still relevant, but hidden in github